### PR TITLE
Fix export of number + pubstate in OFFice XML

### LINKED ISSUE #2509: Office 207 XML export Nunber + Pubstate
TODO: Check export:
http://tex.stackexchange.com/questions/351303/exporting-references-to-ms-word-with-bibtex4word/351452#351452

### PR DESCRIPTION
**Pull request by Siedlerchr**
Fixes #2509
Export/import volumes <-> numberofvolume
Export pubstate field as bibtex only field

- [x] Change in CHANGELOG.md described
- [x] Tests created for changes
~~- [ ] Screenshots added (for bigger UI changes)~~
- [x] Manually tested changed features in running JabRef
- [x] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)
~~- [ ] If you changed the localization: Did you run `gradle localizationUpdate`?~~

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,11 +20,13 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
     - Redesigned journal abbreviations dialog.
     - Redesigned error console.
     - All file dialogs now use the native file selector of the OS. [#1711](https://github.com/JabRef/jabref/issues/1711)
+- The MS-Office XML export now exports the field `volumes` and `pubstate`.
```
Comment (lenhard) [inline CHANGELOG.md]: Minor thing: This should be aligned with the other bullet points.

File: src/test/resources/net/sf/jabref/logic/exporter/MsBibExportFormatTest1.bib [hunk 2]
```diff
@@ -26,6 +25,7 @@ @Unpublished{
   size                        = {1000},
   sourcetype                  = {Interview},
   timestamp                   = {2016.08.19},
+  volumes 					  = {10}
```
Comment (lenhard) [inline src/test/resources/net/sf/jabref/logic/exporter/MsBibExportFormatTest1.bib]: Again, please align.

File: src/test/resources/net/sf/jabref/logic/importer/fileformat/MsBibImporterMsBibFieldsTest.bib [hunk 3]
```diff
@@ -20,6 +19,7 @@ @Article{
   price                       = {10},
   school                      = {Gymnasium Unterrieden},
   size                        = {1000},
+  volumes 					  = {10},
```
Comment (lenhard) [inline src/test/resources/net/sf/jabref/logic/importer/fileformat/MsBibImporterMsBibFieldsTest.bib]: Again, please align.

**Comment by Siedlerchr** [comment]
Hm seems like a problem with spaces and tabs again

Am 03.02.2017 8:42 vorm. schrieb "Jörg Lenhard" <notifications@github.com>:

File: CHANGELOG.md (hunk 1 above)
Comment (Siedlerchr) [inline CHANGELOG.md]: doen

**Pull request merged**